src/NBitcoin/NetworkOptions.cs

```diff
@@ -31,11 +45,53 @@ public NetworkOptions(uint flags = All)
             this.flags = flags;
         }
 
+        /// <summary>
+        /// Get/Set ProofOfStake flags.
+        /// </summary>
+        public bool IsProofOfStake
+        {
+            get
+            {
+                bool isPOS = (this.flags & POS) != 0;
+                // This sanity check will be removed once the static flags are removed
```

Comment:
Maybe just add a TODO here so we can remember ;)

Reply:
Done